source/release_notes/5.0.9-8.rst

```diff
@@ -0,0 +1,36 @@
+.. _PSMDB-5.0.9-8:
+
+================================================================================
+*Percona Server for MongoDB* 5.0.9-8 (2022-06-15)
+================================================================================
+
+:Date: June 15, 2022
+:Installation: :ref:`install`
+
+Percona Server for MongoDB 5.0.9-8 is an enhanced, source available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.9 Community Edition.
+It supports MongoDB 5.0.9 protocols and drivers.
+
+Release Highlights
+==================
+
+* Support of :ref:`multiple KMIP servers <kmip>` adds failover to your data-at-rest encryption setup.
+* The support of encrypted KMIP client certificates and keys through the password setting. 
```

Comment:
Can we expand the value a bit here? So I can now set the password for KMIP certs. What does it give me, as a user? Why should this excite me to update?